contentScript.js

```diff
@@ -12,7 +12,18 @@ const isOnJoinPage = () =>
 const hideSelfButton = () =>
   document
     .evaluate(
-      "//div[following-sibling::span//i[contains(., 'close_fullscreen')]]",
+      "//span[i[contains(., 'close_fullscreen')]]",
+      document,
+      null,
+      XPathResult.ANY_TYPE,
+      null
+    )
+    .iterateNext();
+
+const moreButton = () =>
+  document
+    .evaluate(
+      "//button[i[contains(., 'more_vert')]]",
```

Comment:
The following works for me in both framed and unframed views 😄 

```suggestion
      "//i[contains(., 'auto_awesome')]/following::i[contains(., 'more_vert')]",
```